_includes/video/pb-vm-vjs.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" class="html--no-js">
+
+<head>
+    {% include head--common.html %}
+    <!-- player code -->
+    {% if jekyll.environment == "production" %}
+    <script>
+        window.onetrustLoaded.then(() => {
+            OneTrust.InsertScript("https://vjs.zencdn.net/7.20.2/video.min.js", 'head', null, {}, 'C0003');
```

Comment:
Have you tested this locally? IE removing the production condition. I've found it challenging to queue the ordering of things with this damn OneTrust stuff. It would be good to ensure there isn't an issue here.

Reply:
I'm just asking if you have tested this onetrust code locally to ensure it's going to work in production.